test/integration/xpack-stig.yml

```diff
@@ -0,0 +1,198 @@
+---
+# STIG based xpack provision for inspec hardening compliance
+# Make sure to change the variables below in the vars section
+# and in the kitchen.vagrant.yml file
+
+- import_playbook: roles/elasticsearch/test/integration/xpack.yml
+
+- name: Elasticsearch Xpack STIG
+  hosts: localhost
+  vars:
+    es_conf_dir: "/etc/elasticsearch"
+    es_instance_name: "security_node"
+    es_home: "/usr/share/elasticsearch"
+    inventory_hostname: "localhost"
+    log_aggregation_system: "logagg.site.mil"
+    es_logs_dir: "/var/log/elasticsearch/{{inventory_hostname}}-{{es_instance_name}}/"
+
+  environment:
+    ES_PATH_CONF: "{{es_conf_dir}}"
+  tasks:
+
+    - name: Install necessary packages
+      yum: state=present name={{ item }}
+      with_items:
+        - unzip
+        - openssl
+        - rsyslog
+        - audit
+        - chrony
+        - ntp
+        - aide
+
+    - name: Copy instances.yml
+      template:
+        src: /tmp/kitchen/roles/elasticsearch/templates/instances.yml
```

Comment:
Since this is only used for testing could you move the `instances.yml` into the `test/` directory.